### PR TITLE
Add and list registrants for a meeting

### PR DESCRIPTION
**Pull request by thomjass**
Add the possibility to add a participant to a meeting and to list the registrants of a meeting.

### CODE REVIEW

File: zoomus/components/meeting.py [hunk 1]
```diff
@@ -70,3 +70,14 @@ def delete(self, **kwargs):
...
             "/meetings/{}".format(kwargs.get("id")), params=kwargs
         )
+    
+    def add_registrant(self, **kwargs):
+        util.require_keys(kwargs, "id")
+        return self.post_request(
+            "/meetings/{}/registrants".format(kwargs.get("id")), data=kwargs
+        )
+    
+    def list_registrant(self, **kwargs):
```
Comment (prschmid) [inline zoomus/components/meeting.py:80]: This method should likely be plural --> i.e. `list_registrants`

**Comment by chirag200666** [comment]
@prschmid the review changes were implemented, can you please merge this PR.

**Comment by chirag200666** [comment]
@thomjass can you please add the new functionality in the Readme as well.

**Comment by prschmid** [comment]
Looks like another PR was added after yours that added a couple more tests, etc. Please see https://github.com/prschmid/zoomus/pull/105. I will mark this one as duplicate and close off this PR in favor of the other one. Thank you so much for your and @ducpho 's work!

**Pull request closed without merging**